src/form-workflows.rst

```diff
@@ -0,0 +1,215 @@
+
+* :fa:`external-link` `Example of a single form supporting multiple encounters <https://docs.google.com/spreadsheets/d/1I9kWZOh4lbIR9T35iSC2YxLQeBI6ZApoBp_C7twx5Ec>`_
+
+There are limitations to representing multiple encounter workflows with a single form:
+
+* A single device must be used to perform the whole workflow.
+* There’s no opportunity to clean, monitor or report on data between steps of the workflow.
+* If the workflow needs to be interrupted, finding the correct filled form to edit and the correct question to jump to can be tedious (:ref:`instance name <instance-name>` and :ref:`groups <groups>` help with this).
+* A single long form can be hard to test and troubleshoot.
+
+.. _multiple-encounters-across-different-submissions:
+
+Multiple encounters across different submissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Many workflows can’t be captured by a single form submission because:
+
+* Different steps of the workflow are completed by different people and/or at different times
+* Data is needed between steps for things like reports, monitoring, cleaning, assignment, eligibility determination
+
+Workflows can be split across multiple submissions of the same or different forms. The submissions are linked during analysis by a common entity identifier. The identifier can also optionally be used to :ref:`look up data about the selected entity <using-data-from-previous-encounters>`.
+
+The sections below describe some common tools and patterns for capturing multiple encounters across different submissions. As long as multiple submissions have common data, they can be linked no matter what their structure is so many variations are possible. 
+
+.. _using-barcodes-to-link-encounters:
+
+Using barcodes to link encounters
+"""""""""""""""""""""""""""""""""""
+Barcodes are ideal for uniquely identifying entities because they can be consistently scanned without concern for data entry errors.
+
+* :fa:`external-link` `Example of a form for an initial encounter <https://docs.google.com/spreadsheets/d/1jQrhKJ71oMsfDgyeP5HxNp3HdDjq6yXeHYzulZ8Nq7Y>`_
+* :fa:`external-link` `Example of a form for a follow-up encounter <https://docs.google.com/spreadsheets/d/1UhHpnvqEI_K4UEwDQHN86fR5CQdBI2wX2mou8oyTlOw>`_ 
+
+If barcodes aren't available or practical, you can use other kinds of unique identifiers such as phone numbers or names that will be entered manually. Be careful about identifiers that may not be unique, that may change, or that can be lost.
+
+Once submissions are received, they can be linked through the unique identifier. For example, this can be done `using Excel functions <https://support.microsoft.com/en-us/office/look-up-values-with-vlookup-index-or-match-68297403-7c3c-4150-9e3c-4d348188976b>`_.
```

Comment:
s/Excel functions/Excel's VLOOKUP/

saves me clicking if I already know what VLOOKUP is.